src/hooks/events.ts

```diff
@@ -25,13 +29,24 @@ export interface VenueEventsData {
 export const useVenueEvents: ReactHook<VenueEventsProps, VenueEventsData> = ({
   venueIds,
 }) => {
+  const firebase = useFirebase();
+
   const {
     loading: isEventsLoading,
     error: eventsError,
     value: events = emptyArray,
   } = useAsync(async () => {
-    if (!venueIds) return emptyArray;
+    //load from gs
+    const storage = firebase.storage();
+    const url = await storage
+      .ref()
+      .child(SCHEDULE_LOADFROM_GS)
+      .getDownloadURL();
```

Comment:
Are we not going to need to grant some kind of 'is allowed to read storage' permissions in the rules/etc for this to work? I don't see any changes to `storage.rules` in this PR?

Reply:
check if needed

Reply:
@yarikoptic I don't understand if you're making a note here for yourself, telling me to do something, or something else entirely?

Reply:
I had to update the cors but you don't need any update for storage rules as far as I know. The way it's setup right now works just fine.

Reply:
Actually, I needed to add this to the rule

```
    match /cache/{allPaths=**} {
        allow read
    }
```

I will move it under /assets so we don't have to add this.